docs/reference/misc/sounds.mdx

```diff
@@ -0,0 +1,160 @@
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>1.2</label>
+        </td>
+      </tr>
+      <tr>
+        <td>ALERT</td>
+        <td>
+          High-pitched note pling; used for <a href="/docs/modules/information/broadcasts">Alert broadcast messages</a>.
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>2.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>PORTAL</td>
+        <td>
+          Used for <a href="/docs/modules/mechanics/portals">Portals</a>.
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>SCORE</td>
+        <td>
+          Used when the player enters a <a href="/docs/modules/objectives/scoring#score-boxes">Score Box</a>.
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          OBJECTIVE_FIREWORKS_FAR
+          <br />
+          OBJECTIVE_FIREWORKS_TWINKLE
+        </td>
+        <td>
+          Used for <a href="/docs/modules/objectives/dtm#destroyable-attributes">Destroyables</a> when <label>sparks</label> is enabled.
+        </td>
+        <td>
+          <label>0.75</label>
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>OBJECTIVE_GOOD</td>
+        <td>
+          Used when the player's team completes or captures an objective.
+        </td>
+        <td>
+          <label>0.7</label>
+        </td>
+        <td>
+          <label>2.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>OBJECTIVE_BAD</td>
+        <td>
+          Used when the enemy team completes or captures an objective.
+        </td>
+        <td>
+          <label>0.8</label>
+        </td>
+        <td>
+          <label>0.8</label>
+        </td>
+      </tr>
+      <tr>
+        <td>OBJECTIVE_MODE</td>
+        <td>
+          Used when <a href="/docs/modules/objectives/monument-modes">Monument Modes</a> is triggered.
+        </td>
+        <td>
+          <label>0.15</label>
+        </td>
+        <td>
+          <label>1.2</label>
+        </td>
+      </tr>
+      <tr>
+        <td>DEATH_OWN</td>
+        <td>
+          Used when the player or a teammate dies during a match.
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+      </tr>
+      <tr>
+        <td>DEATH_OTHER</td>
+        <td>
+          Used when an enemy player dies during a match.
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+        <td>
+          <label>1.0</label>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+Copied from: [Kyori Adventure docs - Sound](https://docs.advntr.dev/sound.html)
```

Comment:
We should also add [`Sounds.json`](https://minecraft.wiki/w/Sounds.json#Sound_events) as a reference here. If Adventure/Bukkit has a more specific sound library, that can be used instead. Otherwise, a quick example on how to make a Key using `Sounds.json` from the `Sound File Used` column should be included.

```suggestion
Copied from: [Kyori Adventure docs - Sound](https://docs.advntr.dev/sound.html)
<br/>
List of Sound Keys can be found [here](https://minecraft.wiki/w/Sounds.json#Sound_events).
```

Reply:
This would be a better link, albeit with some exceptions: https://minecraft.wiki/w/Sounds.json/Java_Edition_values_before_1.9

According to zzuf, `fireworks.largeBlast`, `fireworks.largeBlast_far`, and `item.fireCharge.use` cannot be used because of capitalization.